xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -84,11 +84,11 @@ class CourseContentTableViewController: UITableViewController {
         switch segue.identifier! {
         case "ShowVideo":
             let videoView = segue.destination as! VideoViewController
-            videoView.courseItem = courseItem
+            videoView.courseItem = try! CourseItemHelper.getByID(courseItem!.id)
```

Comment:
It should not be possible the get an error here, but I was also thinking about an appropriate error handling. Do you have an idea?